pipeline/id_minter/src/main/scala/weco/pipeline/id_minter/steps/IdentifierGenerator.scala

```diff
@@ -11,6 +11,17 @@ import scala.util.{Failure, Success, Try}
 class IdentifierGenerator(identifiersDao: IdentifiersDao) extends Logging {
   import IdentifiersDao._
 
+  private val conceptSubTypes = List(
```

Comment:
Should this align with: https://github.com/wellcomecollection/catalogue-pipeline/blob/main/common/internal_model/src/main/scala/weco/catalogue/internal_model/work/AbstractRootConcept.scala#L15?

Reply:
Thanks for the explanation, one non-blocking comment that it might be a good idea to add a comment to where the source is copied from (and vice-versa) if they need to be kept in sync.